qtl/number.h

```diff
@@ -6,7 +6,9 @@
 #include <boost/core/demangle.hpp>
 #include <iostream>
 #include <utility>
+#ifndef __APPLE__ // my include paths have gotten messed up. Hopefully the Docker installation will fix this 
```

Comment:
Maybe this would be a better place to put comments like that?
It's more to explain pull changes to contributors than for users of the code,
And should hopefully go away before end users ever see it.
But I'm not sure how these comments get tracked, or how to search or refer to them.
I just discovered the github feature of making these comments, so I'm trying it out to see how it might be useful.